awx/main/tasks.py

```diff
@@ -2291,7 +2291,7 @@ def make_local_copy(project_path, destination_folder, scm_type, scm_revision):
             # force option is necessary because remote refs are not counted, although no information is lost
             git_repo.delete_head(tmp_branch_name, force=True)
         else:
-            copy_tree(project_path, destination_folder)
+            copy_tree(project_path, destination_folder, preserve_symlinks=1)
```

Comment:
nitpick, but I'm guessing we actually want `True` here?

Reply:
Dunno, distutils docs have 1/0. 

Why are we using distutils instead of shutil anyway?